equipmanager.lic

```diff
@@ -127,20 +127,43 @@ class EquipmentManager
   end
 
   def remove_item(item)
-    bput("remove my #{item.short_name}", "You .*#{item.name}", 'The leather gauntlets slide', 'Without any effort', 'you manage to loosen')
+    result = bput("remove my #{item.short_name}", "You .*#{item.name}", 'The leather gauntlets slide', 'Without any effort', 'you manage to loosen', "You'll need both hands free to do that", "then constricts tighter around your")
     waitrt?
-    # If removing item transforms it (e.g. exoskeletal armor => orb) then continue with the transformed item.
-    if item.transforms_to && DRCI.in_hands?(item.transforms_to)
-      item = item_by_desc(item.transforms_to)
-    end
-    if item.tie_to
-      stow_helper("tie my #{item.short_name} to #{item.tie_to}", item.short_name, 'You attach', 'you tie', 'You are a little too busy', 'Your wounds hinder your ability to do that')
-    elsif item.wield
-      stow_helper("sheath my #{item.short_name}", item.short_name, 'Sheathing', 'You sheathe', 'You .* unload', 'You secure your', 'You slip', 'You hang', 'You .* strap')
-    elsif item.container
-      stow_helper("put my #{item.short_name} into #{item.container}", item.short_name, 'You put', 'You should unload', 'You easily strap', 'You secure your')
-    elsif /more room|too long to fit/ =~ bput("stow my #{item.short_name}", 'You put', 'You should unload', 'You easily strap', 'You secure your', 'There isn\'t any more room', 'straps have all been used', 'is too long to fit')
-      bput("wear my #{item.short_name}", 'You ')
+    case result
+    when /then constricts tighter around your/
+      # Items that auto-repair, like exoskeletal armor,
+      # may have a timer on them that prevents you removing them.
+      DRC.message("The #{item.name} is not ready to be removed yet. Try again later.")
+      return false
```

Comment:
Lines 133-137 are new to handle when exoskeletal armor is not ready to be removed. There seems to be a timer once it activates auto-repair where you are not able to remove it for a time. This can last several minutes to hours.